ttax/ops.py

```diff
@@ -291,3 +298,40 @@ def are_batches_broadcastable(tt_a, tt_b):
     else:
       return False
   return True
+
+
+def multiply(a, b):
+  """Returns the result of multiplication of so called
```

Comment:
Not sure if it's a general thing, but our internal python style guide says that the first sentence of docstring should fit in the first line and end with a dot. Plus I wouldn't talk about wrapped TT in a docstrings: users don't need to know about those. Just something like "Elementwise multiplication of two TT objects or number times TT multiplication."